extensions/windows-event-log/wel/WindowsEventLog.cpp

```diff
@@ -118,8 +120,10 @@ void WindowsEventLogMetadataImpl::renderMetadata() {
 }
 
 std::string WindowsEventLogMetadataImpl::getEventData(EVT_FORMAT_MESSAGE_FLAGS flags) const {
-  LPWSTR string_buffer = NULL;
-  DWORD string_buffer_size = 0;
+  DWORD string_buffer_size = 4096;
```

Comment:
I don't know why Microsoft decided to call this `BufferSize` but store the number of characters in it, but we could give it a better name, eg. `num_chars_in_buffer`.

Reply:
done